### DISCUSSION TITLE
Consider supporting SenseVoice in transcription task

### DISCUSSION THREAD

**Discussion by MaleicAcid** [zh-plus/openlrc#53] [New Feature]
SenseVoice released by Alibaba Group claims to be better than the Whisper model in transcription tasks and "requires only 70ms to process 10 seconds of audio, which is 15 times faster than Whisper-Large."

ref@https://github.com/FunAudioLLM/SenseVoice/blob/main/README_zh.md

**Comment by zh-plus** [comment]
The accuracy of the currently released SenseVoice-small is significantly lower than that of Whisper-large-v3, so it will not be considered as a transcription engine for now. I will test SenseVoice-Large after its release.
